src/toasts/toasts.css

```diff
@@ -0,0 +1,40 @@
+.omrs-toast {
+  visibility: hidden;
```

Comment:
Why is this hidden?

Reply:
It was from the previous implementation. I removed that and it works. Needs to be styled and to add options for button or without close button. A question. Greg designed a retry button. Is this in our scope or just display "Error" in a toast with a close button is all we want?

Reply:
For now, don't worry about retries.

> It was from the previous implementation. I removed that and it works. 

The `visibility: hidden;` is not removed. Maybe you forgot to push that code?

Reply:
I didn't push the changes.